src/calculateRentalCost.js

```diff
@@ -20,7 +20,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const discountKlein = 20;
+  const discountGross = 50;
+
+  if (days < 3) {
+    return days * cost;
+  } else if ((days >= 3) && (days < 7)) {
+    return days * cost - discountKlein;
```

Comment:
Добавь скобочки для читаемости
```suggestion
    return (days * cost) - discountKlein;
```